etc/init.d/zfs.in

```diff
@@ -0,0 +1,74 @@
+# ZoL userland configuration.
+
+# Run `zfs mount -a` during system start?
+ZFS_MOUNT='yes'
+
+# Run `zfs unmount -a` during system stop?
+ZFS_UNMOUNT='yes'
+
+# Run `zfs share -a` during system start?
+# nb: The shareiscsi, sharenfs, and sharesmb dataset properties.
+ZFS_SHARE='yes'
```

Comment:
@Alexqw From what I read about that last comment, everything and everyone did exactly as intended.

<code>dpkg</code> saw that a file tagged as a config file was modified and notified you, the administrator, about this. YOU did what you where asked to do and checked the differences and rejected that [and/or more] part(s). As you should, because you didn't need or wanted that.

This is EXACTLY the reason why it's tagged as a config file - for your to decide if the [new] defaults is valid or not. _I_ consider this new default the best for the most amount of people. Some people will disagree, as they should. But since it's a config file, it is their _responsibility_ to modify it according to their needs and requirements and being a config file, it is protected from any changes I, the packager, do and provide for you.